src/content/en/2020/mobile-web.md

```diff
@@ -7,20 +7,20 @@ description: Mobile Web chapter of the 2020 Web Almanac covering page loading, t
 authors: [spanicker, mdiblasio]
 reviewers: [obto]
 analysts: [obto]
-editors: []
+editors: [exterkamp]
 translators: []
-spanicker_bio: Shubhie Panicker is the engineering lead for Chrome’s engagement in the web framework ecosystem, where she collaborates with open source tools, frameworks and communities. As a member of Chrome’s Web Platform team she has worked on web standards and chromium’s implementation for several web performance APIs. Prior to Chrome, she worked on infrastructure and web frameworks for Google products like Search, Google Photos etc.
+spanicker_bio: Shubhie Panicker is the engineering lead for Chrome's engagement in the web framework ecosystem, where she collaborates with open source tools, frameworks and communities. As a member of Chrome's Web Platform team she has worked on web standards and chromium's implementation for several web performance APIs. Prior to Chrome, she worked on infrastructure and web frameworks for Google products like Search, Google Photos etc.
 mdiblasio_bio: Michael DiBlasio is Web Ecosystems Consultant at Google. He focuses on helping to improve the health of the web ecosystem and to ensure the web is commercially viable for creators and partners. He works closely with strategic retailers to adopt new modern web technologies and improve the quality of existing web experiences. Prior to Google, Michael was a consultant at IBM.
 discuss: 2048
 results: https://docs.google.com/spreadsheets/d/1DGLY7UEWOlDL5_2dtS_j2eqMjiV-Rw5Fe2y6K6-ULvM/
 queries: 12_Mobile_Web
 featured_quote: Mobile Web has grown explosively in the last decade and is now the primary way many people experience the web. In spite of this, engagement and online sales still lag behind desktop. In this chapter, we take a look at recent trends on the mobile web and analyze why user journeys are often difficult to complete.
 featured_stat_1: 41.20%
-featured_stat_label_1: % pages with improperly sized images
+featured_stat_label_1: Percent of pages with improperly sized images
 featured_stat_2: 60.1%
-featured_stat_label_2: % eCommerce landing pages that are missing presence of search input
+featured_stat_label_2: Percent of eCommerce landing pages that are missing presence of search input
 featured_stat_3: 2.6s
-featured_stat_label_3: p75 LCP for mobile
+featured_stat_label_3: 75th percentile LCP for mobile
```

Comment:
@spanicker should we qualify that this p75 is for all sites in the dataset? It is kinda ambiguous what this 2.6s is "for".

Reply:
yes sounds good to add "all sites in dataset"

Reply:
I think it's fine and don't think this is needed.